### ISSUE TITLE
Gauge option not editable (33.7.3 and 34.1) 

### ISSUE THREAD

**Issue by prigal** [metabase/metabase#11775]
**Describe the bug**
Impossible to adjust options of a Gauge (egs : color and scale)

**Logs**

```
Warning: can't clean query without metadata! 2 app-main.bundle.js:5:40881
TypeError: t is null
app-main.bundle.js:5:2068276
    value https://********/app/dist/app-main.bundle.js?8902e0cb00fa4456f7bb:5
    value https://********/app/dist/app-main.bundle.js?8902e0cb00fa4456f7bb:5
    value self-hosted:871
    c https://********/app/dist/vendor.bundle.js?8902e0cb00fa4456f7bb:1
Error: Minified React error #44; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=44 for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 3 vendor.bundle.js:1:285932
    exports https://********/app/dist/vendor.bundle.js?8902e0cb00fa4456f7bb:1
    exports https://********/app/dist/vendor.bundle.js?8902e0cb00fa4456f7bb:1
    value https://********/app/dist/app-main.bundle.js?8902e0cb00fa4456f7bb:5
    _updateSize https://********/app/dist/app-main.bundle.js?8902e0cb00fa4456f7bb:5
```

**To Reproduce**
Check gif.

**Expected behavior**
I should be allowed to edit scale, colors and other gauge options.

**Screenshots**
![bug-gauge](https://user-images.githubusercontent.com/895267/72874626-c1d5fc00-3cf2-11ea-99a9-c41b0342bde4.gif)

**Information about your Metabase Installation:**

```
{
  "browser-info": {
    "language": "fr-FR",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.117 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.4+11-post-Debian-1bpo91",
    "java.vendor": "Debian",
    "java.vendor.url": "https://tracker.debian.org/openjdk-11",
    "java.version": "11.0.4",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.4+11-post-Debian-1bpo91",
    "os.name": "Linux",
    "os.version": "4.15.18-4-pve",
    "user.language": "en",
    "user.timezone": "Europe/Paris"
  },
  "metabase-info": {
    "databases": [
      "mysql",
      "googleanalytics"
    ],
    "hosting-env": "unknown",
    "application-database": "mysql",
    "run-mode": "prod",
    "version": {
      "date": "2020-01-13",
      "tag": "v0.34.1",
      "branch": "release-0.34.x",
      "hash": "265695c"
    },
    "settings": {
      "report-timezone": "Europe/Brussels"
    }
  }
}
```

**Severity**
It's really annoying, sometime blocking as gauge is no more meaningfull.

**Additional context**
-

**Comment by dacort** [comment]
Hi @prigal I can reproduce this as well, but slightly different.

As a temporary workaround - if you switch the visualization type to something else and then back to gauge, does the "Display" tab show up where you can change the colors?

In my repro, when I first click the "gauge" button, the Display options pop up, but then if I click the Formatting button, I can't click the "back" button as you noted in your gif (thank you for that, by the way).

**Comment by dacort** [comment]
Looks like this was introduced in v0.33.5.1.

**Comment by prigal** [comment]
> As a temporary workaround - if you switch the visualization type to something else and then back to gauge, does the "Display" tab show up where you can change the colors?

No, I have tried all ticks I could imagine :) but impossible to change colors. At the beginning I thought it came from the fact that I did not add an alias to my field, but no, even with a simple new question, problem is the same. Reproductible.

**Comment by mafouto** [comment]
I found out if you change your language settings to English you can access the options

**Portuguese: can't see the options**
<img width="1415" alt="Screenshot 2020-01-29 at 14 29 55" src="https://user-images.githubusercontent.com/30630335/73363456-0f310b00-42a9-11ea-8de9-90c850336f0c.png">

**English: options are there**
<img width="1426" alt="Screenshot 2020-01-29 at 14 36 59" src="https://user-images.githubusercontent.com/30630335/73363493-240d9e80-42a9-11ea-8395-f55f8d814f0a.png">

It looks similar to me because the original gif was also in a language other than English. I saw the problem in Portuguese and German as well.

**Comment by flamber** [comment]
Fixed by PR #11833
